src/Framework/Environment.php

```diff
@@ -62,7 +62,7 @@ public static function setupColors()
 	{
 		self::$useColors = getenv(self::COLORS) !== FALSE
 			? (bool) getenv(self::COLORS)
-			: (PHP_SAPI === 'cli' && ((function_exists('posix_isatty') && posix_isatty(STDOUT))
+			: ((PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg') && ((function_exists('posix_isatty') && posix_isatty(STDOUT))
```

Comment:
Is `PHP_SAPI === 'phpdbg'` correct?  PHP_SAPI is 'cli' when argument `-qrrb -S cli` is used.